Sources/WishKit/SwiftUI/CreateWishView.swift

```diff
@@ -209,20 +212,32 @@ struct CreateWishView: View {
             descriptionText = String(descriptionText.prefix(descriptionLimit))
         }
 
-        // Enable/Disable button
-        isButtonDisabled = titleText.isEmpty || descriptionText.isEmpty
+        self.checkButtonStatus()
+    }
+
+    private func handleEmailChange() {
+        guard !self.emailText.isEmpty else { return }
+        let emailValidationRegex = "[A-Z0-9a-z._%+-]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,64}"
+        let emailValidationPredicate = NSPredicate(format: "SELF MATCHES %@", emailValidationRegex)
+        self.isEmailValid = emailValidationPredicate.evaluate(with: self.emailText)
```

Comment:
There seem to be [better ways](https://www.swiftbysundell.com/articles/validating-email-addresses/) of validating e-mail addresses. Your call.